docs/_docs/About us/Member-Directory.md

```diff
@@ -8,165 +8,71 @@ What's in this page:
+The executive team consists of one person.
 
-* President: [Minh Hoang](#minh-hoang)
-* Vice-president: [Carol Martins](#carol-martins)
-* Vice-president: [Inês Mendes](#ines-mendes)
+* President: **Minh Hoang**
 
-### Scope of AORs (Areas of Responsibility)
-
-In the case that an AOR Primary Enabler is not able to decide or resolve an urgent matter in their AOR, or when a Primary Enabler needs some kind of decision from the Executive Team, the go-to persons for each of the AORs are the following:
-
-* [Minh Hoang](#minh-hoang): **Volunteers**, **Partnerships**, **Jobs**, **Documentation**, **Admissions**, **Curriculum Development**
-* [Carol Martins](#carol-martins): **Marketing**, **Operations**, **Community**, **Legal**, **Finance**
-* [Inês Mendes](#ines-mendes): **Teaching**, **Quality Assurance**, **Student Success**, **Dev-Ops** 
-
-## Organizational structure
-
-The first person listed in each Area of Reponsibility (AOR) below is the [primary enabler](../LDSA-Areas-of-Responsibility/#primary-enabler).
+## All LDSA Staff
 
 ### LDSA AORs
 
 | Area of Responsibility | People |
 |-|-|
-|[Curriculum Development](../LDSA-Areas-of-Responsibility/#curriculum-development)| [**Francisco Fonseca**](#francisco-fonsenca) | 
-|[Teaching](../LDSA-Areas-of-Responsibility/#teaching)| [**Catarina Silva**](#catarina-silva) <br> [List of instructors](#ldssa-instructors) |
-|[Operations](../LDSA-Areas-of-Responsibility/#operations)|[**Silvana Alves**](#silvana-alves) |
-|[Marketing and Communications](../LDSA-Areas-of-Responsibility/#marketing-and-communications)| [**Pedro Ved**](#pedro-ved)|
-|[Partnerships](../LDSA-Areas-of-Responsibility/#partnerships)| [**Daniel Neves**](#daniel-neves)|
-|[Student Success](../LDSA-Areas-of-Responsibility/#student-success)| [**Mária Hanulová**](#Mária-Hanulová) |
-|[Community](../LDSA-Areas-of-Responsibility/#community)| [**Gonçalo Nunes**](#gonçalo-nunes)|
-|[Dev-Ops](../LDSA-Areas-of-Responsibility/#dev-ops)| [**Miguel Dias**](#miguel-dias) |
-|[Quality Assurance](../LDSA-Areas-of-Responsibility/#quality-assurance)| [**Ana Rita Carvalho**](#ana-rita-carvalho) |
-|[Finance](../LDSA-Areas-of-Responsibility/#finance)| [**Maria Dominguez (Chi)**](#maria-dominguez-chi) |
-|[Jobs](../LDSA-Areas-of-Responsibility/#jobs)| [**Telmo Felgueira**](#telmo-felgueira)|
-|Admissions| [**Fábio Cruz**](#fabio-cruz) |
-|[Documentation](../LDSA-Areas-of-Responsibility/#documentation)| [**Juliana Rodrigues**](#juliana-rodrigues) |
-|[Volunteers](../LDSA-Areas-of-Responsibility/#volunteers)| [Gustavo Fonseca](#gustavo-fonseca)|
-
-#### Prep Course
+|[Teaching](../LDSA-Areas-of-Responsibility/#teaching)| **Maria Hanulova** |
```

Comment:
I have two accents in the name: Mária Hanulová

Reply:
Sorry :(
Good that you're reviewing!